src/display/canvas.js

```diff
@@ -3051,6 +3054,13 @@ class CanvasGraphics {
     const ctx = this.ctx;
 
     this.save();
+
+    // The filter, if any, will be applied in applyTransferMapsToBitmap.
+    // It must be applied to the image before rescaling else some artifacts
+    // could appear.
+    // The final restore will reset it to its value.
+    ctx.filter = "none";
```

Comment:
You might want to simply ignore Node.js here, to prevent any possible issue?
```suggestion
    if (!isNodeJS) {
      // The filter, if any, will be applied in applyTransferMapsToBitmap.
      // It must be applied to the image before rescaling else some artifacts
      // could appear.
      // The final restore will reset it to its value.
      ctx.filter = "none";
    }
```